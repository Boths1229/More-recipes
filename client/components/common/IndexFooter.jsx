import React from 'react';
import { Link } from 'react-router-dom';

/**
 * @description render the footer for landing page
 *
 * @returns {JSX} return JSX
 */
const IndexFooter = () => ((
  <footer className="container page-footer">
    <div className="row">
      <div className="col-sm-12">
        <div className="divider" />
      </div>
    </div>
    <div className="row">
      <div className="col-sm-6 col-md-4 rfloat">
        <ul className="pt-2">
          <li>
            <Link to="">
              <i className="fa fa-google-plus" />
            </Link>
          </li>
          <li>
            <Link to="https://github.com/kensanni/More-recipes">
              <i className="fa fa-github" />
            </Link>
          </li>
          <li>
            <Link to="">
              <i className="fa fa-facebook" />
            </Link>
          </li>
          <li>
            <Link to="">
              <i className="fa fa-twitter" />
            </Link>
          </li>
        </ul>
      </div>
      <div className="col-sm-6 col-md-8 left">
        <div className="copyright">
          <small>
            More-Recipes Â© 2017,
            All Rights Reserved,
            Design &amp; Developed By: Sanni Kehinde
          </small>
        </div>
      </div>
    </div>
  </footer>
));

export default IndexFooter;
