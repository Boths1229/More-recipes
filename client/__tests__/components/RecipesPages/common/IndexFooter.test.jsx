import React from 'react';
import { shallow } from 'enzyme';
import IndexFooter from '../../../../components/common/IndexFooter';

describe('IndexFooter component', () => {
  it('matches snapshot', () => {
    const output = shallow(<IndexFooter />);
    expect(output).toMatchSnapshot();
  });
});
