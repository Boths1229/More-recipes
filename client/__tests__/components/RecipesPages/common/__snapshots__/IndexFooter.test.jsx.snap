// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexFooter component matches snapshot 1`] = `
<footer
  className="container page-footer"
>
  <div
    className="row"
  >
    <div
      className="col-sm-12"
    >
      <div
        className="divider"
      />
    </div>
  </div>
  <div
    className="row"
  >
    <div
      className="col-sm-6 col-md-4 rfloat"
    >
      <ul
        className="pt-2"
      >
        <li>
          <a
            href=""
          >
            <i
              className="fa fa-google-plus"
            />
          </a>
        </li>
        <li>
          <a
            href="https://github.com/kensanni/More-recipes"
          >
            <i
              className="fa fa-github"
            />
          </a>
        </li>
        <li>
          <a
            href=""
          >
            <i
              className="fa fa-facebook"
            />
          </a>
        </li>
        <li>
          <a
            href=""
          >
            <i
              className="fa fa-twitter"
            />
          </a>
        </li>
      </ul>
    </div>
    <div
      className="col-sm-6 col-md-8 left"
    >
      <div
        className="copyright"
      >
        <small>
          More-Recipes Â© 2017, All Rights Reserved, Design & Developed By: Sanni Kehinde
        </small>
      </div>
    </div>
  </div>
</footer>
`;
